src/include/iscsi-client/dialogs.rb

```diff
@@ -272,13 +272,13 @@ def initialize_iscsi_client_dialogs(include_target)
               _("No Authentication"),
               true,
               VBox(
-                Left(Label(_("Incoming Authentication"))),
+                Left(Label(_("Outgoing Authentication"))),
                 HBox(
                   HWeight(2, TextEntry(Id(:user_in), _("Username"))),
```

Comment:
Thanks @ancorgs and @cwh42 , I am not a yast or Ruby expert,  but I understand what you mean. I have some work to do this week, so that may take some time correct  it.